### PR TITLE
Add GlusterFS server configuration guide

### PR DESCRIPTION
**Pull request by Sherlock113**
Signed-off-by: Sherlock113 <sherlockxu@yunify.com>

### CODE REVIEW

File: content/en/docs/api-reference/storage-system-installation/glusterfs-server.md [hunk 1]
```diff
@@ -0,0 +1,510 @@
...
+| server1  | 192.168.0.2 | Ubuntu 16.04, 4 Cores, 4 GB Memory | /dev/vdc 100 GB, /dev/vdd 300 GB |
+| server2  | 192.168.0.3 | Ubuntu 16.04, 4 Cores, 4 GB Memory | /dev/vdc 100 GB, /dev/vdd 300 GB |
+| server3  | 192.168.0.4 | Ubuntu 16.04, 4 Cores, 4 GB Memory | /dev/vdc 100 GB, /dev/vdd 300 GB |
+
+{{< notice note >}}
+
+- Heketi will be installed on `server1`, which provides a RESTful management interface to manage the lifecycle of GlusterFS volumes. You can install it on a separate machine as well.
+
+- Attach more block storage disks to your server machine if you need more storage space.
+- Data will be saved to `/dev/vdd` (block device), which must be original without partitioning or formatting.
```
Comment (stoneshi-yunify) [inline content/en/docs/api-reference/storage-system-installation/glusterfs-server.md:30]: why above /dev/vdc was not used? if it is not used, it should not be mentioned.

File: content/en/docs/api-reference/storage-system-installation/glusterfs-server.md [hunk 2]
```diff
@@ -0,0 +1,510 @@
...
+   State: Peer in Cluster (Connected)
+   
+   Hostname: server3
+   Uuid: 9bd733e4-96d4-49d5-8958-6c947a2b4fa6
+   State: Peer in Cluster (Connected)
+   ```
+
+## Install Heketi
+
+As GlusterFS itself does not provide a way for API calls, you can install [Heketi](https://github.com/heketi/heketi) to manage the lifecycle of GlusterFS volumes with a RESTful API for Kubernetes calls. In this way, your Kubernetes cluster can dynamically provision GlusterFS volumes. Heketi v7.0.0 will be installed in this example. For more information about available Heketi versions, see its [Release Page](https://github.com/heketi/heketi/releases/).
```
Comment (stoneshi-yunify) [inline content/en/docs/api-reference/storage-system-installation/glusterfs-server.md:207]: Should explicitly say which server should be operated with following content

Comment (Sherlock113) [reply content/en/docs/api-reference/storage-system-installation/glusterfs-server.md:208]: Changed.

**Review by stoneshi-yunify — APPROVED** [review]
/lgtm

**Pull request merged**